### PR TITLE
Upgrade groovy and kotlin processors

### PR DESCRIPTION
**Pull request by alien11689**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -128,15 +128,17 @@ dependencies {
...
+    compile 'com.github.shyiko.ktlint:ktlint-ruleset-standard:0.31.0'
 
     // detekt
-    compile 'io.gitlab.arturbosch.detekt:detekt-core:1.0.0.RC3'
-    compile 'io.gitlab.arturbosch.detekt:detekt-rules:1.0.0.RC3'
+    compile 'io.gitlab.arturbosch.detekt:detekt-core:1.0.0-RC14'
+    compile 'io.gitlab.arturbosch.detekt:detekt-rules:1.0.0-RC14'
     // dependency only for obtaining default configuration
-    compile 'io.gitlab.arturbosch.detekt:detekt-cli:1.0.0.RC3'
+    compile 'io.gitlab.arturbosch.detekt:detekt-cli:1.0.0-RC14'
```
Comment (SpOOnman) [inline build.gradle]: Nice updates overall!

File: src/main/java/pl/touk/sputnik/processor/detekt/DetektProcessor.java [hunk 2]
```diff
@@ -79,9 +82,19 @@ private Detektor buildDetector(String commonPath) {
...
+        ProcessingSettings processingSettings = new ProcessingSettings(
+                FileSystems.getDefault().getPath(commonPath),
+                config,
+                new ArrayList<PathFilter>(),
+                false,
+                false,
+                new ArrayList<Path>(),
+                executor,
+                System.out,
+                System.err
```
Comment (SpOOnman) [inline src/main/java/pl/touk/sputnik/processor/detekt/DetektProcessor.java]: Why should we pass these streams here? What are they for? Shouldn't Sputnik gather this output and tunnel it somewhere to comments?

Comment (alien11689) [reply src/main/java/pl/touk/sputnik/processor/detekt/DetektProcessor.java]: I really don't know and I am not sure that i would like to know...
I will add dedicated stream

File: src/main/java/pl/touk/sputnik/processor/detekt/DetektProcessor.java [hunk 3]
```diff
@@ -35,22 +40,32 @@
 
     private final Configuration configuration;
 
+    private final ExecutorService executor = ForkJoinPool.commonPool();
+
+    private static PrintStream printStream;
```
Comment (SpOOnman) [inline src/main/java/pl/touk/sputnik/processor/detekt/DetektProcessor.java]: Would you mind moving it to instance field instead of a static one?

Comment (alien11689) [reply src/main/java/pl/touk/sputnik/processor/detekt/DetektProcessor.java]: Done

Comment (SpOOnman) [reply src/main/java/pl/touk/sputnik/processor/detekt/DetektProcessor.java]: Thanks

**Pull request merged**